src/System.Net.Http/src/System/Net/Http/HttpRequestException.cs

```diff
@@ -31,10 +31,17 @@ public HttpRequestException(string message, Exception inner)
 
         // This constructor is used internally to indicate that a request was not successfully sent due to an IOException,
         // and the exception occurred early enough so that the request may be retried on another connection.
-        internal HttpRequestException(string message, Exception inner, bool allowRetry)
+        internal HttpRequestException(string message, Exception inner, RequestRetryType allowRetry)
```

Comment:
I'll be pedantic and mention that this breaks reflection backward compatibility, but I'm assuming it's intentional.

Reply:
It's an internal ctor, so I'm not worried about this. Are you using this ctor in stress testing?

Reply:
Far from it. It's an internal constructor of a public type and there might be customers out there whose reflection depends on the old signature.

Reply:
> there might be customers out there whose reflection depends on the old signature.

We do not worry about reflection to access internal/private members.  If someone's doing that and they break, they will need to change their code.